src/js/components/views/ProblemList.js

```diff
@@ -76,7 +77,11 @@ const ProblemList = ({
             !filters.grade ||
             (grade <= filters.grade.lte && grade >= filters.grade.gte),
         )
-        .map(({ uuid, holds, name, grade, author }) => (
+        .filter(
```

Comment:
Remove. These are still a WIP, the intention is to limit the problems you see on screen e.g between 6b and 7a, starting with `Emoji` etc.